tests/Feature/Repository/TaskRepositoryTest.php

```diff
@@ -30,15 +34,106 @@ public function testGetUserTaskIsSuccessful(): void
         $task->setAttribute('status','pending');
         $task->user()->associate($user);
         $task->save();
-        $task2=new Task();
-        $task2->setAttribute('name','task2');
-        $task2->setAttribute('description','this is task2');
-        $task2->setAttribute('status','pending');
-        $task2->user()->associate($user);
-        $task2->save();
+    
+        $expected = [
+            $task->toArray()
+        ];
+        $repository = new TaskRepository();  
+        $result = $repository->getUserTask($user);
+        self::assertEquals($expected, $result->toArray());
```

Comment:
Failed asserting that two arrays are equal.
 -          'table' => null
 +         'table' => 'users'
  -          'wasRecentlyCreated' => true
  +         'wasRecentlyCreated' => false
   -          'id' => 1
  +           'id' => '1'
  +           'email_verified_at' => null
  +           'remember_token' => null